src/sentry/features/flags/permanent.py

```diff
@@ -0,0 +1,64 @@
+    manager.add("organizations:discover-basic", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:discover-query", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:dynamic-sampling", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:event-attachments", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:global-views", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:incidents", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-alert-rule", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-chat-unfurl", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-codeowners", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-event-hooks", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-incident-management", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-issue-basic", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-issue-sync", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-stacktrace-link", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:integrations-ticket-rules", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:performance-view", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:relay", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:session-replay", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:sso-basic", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:sso-saml2", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:team-insights", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    # NOTE: Don't add features down here! Sort them alphabetically! The order
+    #       features are registered is not important.
+
+    manager.add("projects:custom-inbound-filters", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:data-forwarding", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:discard-groups", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:rate-limits", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:servicehooks", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    # NOTE: Don't add features down here! Sort them alphabetically! The order
```

Comment:
asking engineers to do this without tooling feels burdensome.

Reply:
Does your editor not have a line sort feature?

Reply:
wow I sound grumpy

Reply:
i don't know, i don't sort stuff 😭